vignettes/tcplfit2-vignette.Rmd

```diff
@@ -191,10 +191,11 @@ The input for this example comes from the ACEA_AR assay. Data from the assay com
 * clowder_uid = clowder unique id for source files
 * git_hash = hash key for pre-processing scripts
 
-```{r example4_init, fig.height = 6, fig.width = 7, message=FALSE, warning = FALSE}
+```{r example4_init, fig.height = 6, fig.width = 7, message=FALSE, warning = FALSE,echo=-4}
```

Comment:
What does echo = -4 do? I thought 'echo' must be TRUE or FALSE.

Reply:
-4 means exclude the 4th expression (as the user does not actually need to set this during execution, only for CRAN) https://yihui.org/knitr/options/#text-output

Reply:
Okay, thank you for providing this information and clarification.  Resolving this conflict and approving PR.